streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -98,7 +97,7 @@
     private int testNumber = 0;
 
     @Before
-    public void createTopics() throws InterruptedException {
+    public void createTopics() throws Exception {
```

Comment:
@mjsax justified this in other PRs... Since the exceptions are unexpected, and they would cause the tests to fail if they happened anyway, detailed throws declarations just pollute the test method declarations.